examples/anthos-service-mesh-multicluster/README.md

```diff
@@ -0,0 +1,167 @@
+
+Also, in order for both clusters to access the service mesh (Istiod) and service deployed on the other cluster, we need to do the following:
+- The pod CIDR range of one cluster must be added to the "_Master Auth Network_" of the other cluster. This enables one cluster to ping _istiod_ on the other cluster. 
+- A firewall need to be open for one cluster's pod CIDR to access the service port on the other cluster. In this sample, it is port 5000 used by the HelloWord testing application. Because the invocation of service is bidirection in HelloWorld testing application, we will add firewall rule for each direction. 
+
+The infrastruction used in this sample is coded in Terraform scripts. The ASM installation steps are coded in Shell script.     
+
+## Prerequisites
+
+As mentioned in [this document](https://cloud.google.com/service-mesh/docs/gke-install-multi-cluster), there are several prerequisites. 
+
+This guide assumes that you have:
+
+- [A Cloud project](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+- [A Cloud Billing account](https://cloud.google.com/billing/docs/how-to/manage-billing-account).
+
+Also, multi-cluster configuration has these requirements for the clusters in it:
+
+- All clusters must be on the same network. 
+- If you join clusters that are not in the same project, they must be installed using the asm-gcp-multiproject profile and the clusters must be in a shared VPC configuration together on the same network. In addition, we recommend that you have one project to host the shared VPC, and two service projects for creating clusters. For more information, see [Setting up clusters with Shared VPC](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-shared-vpc).
+
+In this sample, we create two private clusters in different subnets of the same VPC in the same project, and enable clusters to communicate to each other's API server.   
+
+## How to set up and run this sample
+
+### Build Infrastructure
+
+1. Create a GCP project.
+2. Create a VPC in GCP project.
+3. Create a subnet in the VPC. 
+4. Create a VM in the subnet. This will be the bastion server to simulate an intranet access to GKE clusters.  
+5. SSH onto this bastion server.
+6. Make sure you have the following tools installed:
+- The Cloud SDK (the gcloud command-line tool)
+- The standard command-line tools: awk, curl, grep, sed, sha256sum, and tr
+- git
+- kpt
+- kubectl
+- jq
+
+7. Set up Git on your machine, then clone this Github sample onto your machine. 
+
+8. Set up [Terraform](https://learn.hashicorp.com/terraform/getting-started/install.html) on your machine, so you will be able to build infrastructure. 
+
+9. Update the corresponding parameters for your project. 
+
+- In vars.sh, check to see whether you need to update CLUSTER1_LOCATION,CLUSTER1_CLUSTER_NAME, CLUSTER1_CLUSTER_CTX, CLUSTER2_LOCATION, CLUSTER2_CLUSTER_NAME, CLUSTER2_CLUSTER_CTX.
```

Comment:
```suggestion
- In vars.sh, check to see whether you need to update CLUSTER1_LOCATION,CLUSTER1_CLUSTER_NAME, CLUSTER1_CLUSTER_CTX, CLUSTER2_LOCATION, CLUSTER2_CLUSTER_NAME, CLUSTER2_CLUSTER_CTX.
```
```suggestion
- In ``vars.sh``, check to see whether you need to update `CLUSTER1_LOCATION`,`CLUSTER1_CLUSTER_NAME`, `CLUSTER1_CLUSTER_CTX`, `CLUSTER2_LOCATION`, `CLUSTER2_CLUSTER_NAME`, `CLUSTER2_CLUSTER_CTX`.
```
See: https://developers.google.com/style/code-in-text https://developers.google.com/style/filenames

Reply:
Added README formatting changes manually